### DISCUSSION TITLE
Question about language server process

### DISCUSSION THREAD

**Discussion by minyoungyang** [eclipse-theia/theia#7995] [General] [answered]
I am using Theia to create IDE that works in cloud environment.
However, when several users were connected, it was found that language server process increased by the number of users. In my opinion, if a certain number of users use the same language server process, it will increase the cloud efficiency. 

Is there a reason you designed this? I would like to see any related data or documents. 

And, I want to know if I can change multiple users to use one language server. Thank you.

**Comment by akosyakov** [comment, marked as answer]
Because LSP is not designed for multiple users. Each user has own dirty text documents.
